src/App.js

```diff
@@ -43,7 +43,7 @@ class App extends Component {
       return <AppPlaceholder />;
     }
 
-    if (!isAdmin && insights.chrome.getApp() === 'rbac') {
+    if (!isAdmin) {
```

Comment:
Won't this cause that `My user access` page will be also denied?

Reply:
MUA is should only be available in ci-beta and it's going through a redesign anyway